### PR TITLE
Implement basic student GUI

### PR DESCRIPTION
**Pull request by Ansel-Ch**
image
This is how it looks at the moment. Will make it resemble the wireframe we created by the next commit 👍

### CODE REVIEW

**Comment by Ansel-Ch** [comment]
image
Here's the latest image.

**Comment by Ansel-Ch** [comment]
> image Here's the latest image.

I made it so it supports streching.

**Comment by Ansel-Ch** [comment]
Don't merge this yet as Im still waiting for the `Assignment` to be finalised.

**Comment by Ansel-Ch** [comment]
image
Here's the updated CLI.

**Comment by Ansel-Ch** [comment]
image
Latest image with assignments.

File: src/main/java/seedu/address/model/assignment/Assignment.java [hunk 1]
```diff
@@ -83,12 +83,12 @@ public Grade getGrade() {
     @Override
     public String toString() {
         return new ToStringBuilder(this)
-                .add("name", assignmentName)
-                .add("deadline", deadline)
-                .add("submission status", submissionStatus)
-                .add("grading status", gradingStatus)
-                .add("grade", grade)
-                .toString();
+                 .add("name", assignmentName)
```
Comment (jerrelllee) [inline src/main/java/seedu/address/model/assignment/Assignment.java:86]: Is this intentional

**Review by gnoossk — APPROVED** [review]

**Pull request merged**